ui/src/dashboards/actions/thunks.ts

```diff
@@ -363,6 +370,10 @@ export const getDashboard = (dashboardID: string) => async (
     dispatch(creators.setDashboard(dashboardID, RemoteDataState.Done, normDash))
     dispatch(updateTimeRangeFromQueryParams(dashboardID))
   } catch (error) {
+    if (error.name === 'AbortError') {
```

Comment:
we allso check for `'CancellationError'` in some handlers. not sure if we need that here, but wanted to bring it up